docs/reference/linstorsatellite.md

```diff
@@ -0,0 +1,57 @@
+# `LinstorSatellite`
+
+This resource controls the state of a LINSTOR® satellite.
+
+**NOTE:** This resource is not intended to be changed directly, instead it is created by the Piraeus Operator
+by merging all matching [`LinstorSatelliteConfiguration`](./linstorsatelliteconfiguration.md).
+
+## `.spec`
+
+Holds the desired state the satellite.
+
+### `.spec.repository`
+
+Holds the default image registry to use for all Piraeus images. Inherited from
+[`LinstorCluster`](./linstorcluster.md#specrepository).
+
+If empty (the default), the operator will use `quay.io/piraeusdatastore`.
+
+### `.spec.clusterRef`
+
+Holds a reference to the [`LinstorCluster`](./linstorcluster.md) that controls this satellite.
+
+### `.spec.storagePools`
+
+Holds the storage pools to configure on the node. Inherited from matching
+[`LinstorSatelliteConfigurations`](./linstorsatelliteconfiguration.md#specstoragepools).
```

Comment:
In the intro section you wrote `LinstorSatelliteConfiguration`, here `LinstorSatelliteConfigurations`. We should be consistent. I would use either "`LinstorSatelliteConfiguration`" or "`LinstorSatelliteConfiguration`s", not "`LinstorSatelliteConfigurations`".